src/dashboards/Poster/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Images from '../../../../assets/images/app_images.webp'
+import './Footer.scss'
+
+const Footer = (): JSX.Element => (
```

Comment:
Kan være at denne burde hete PosterFooter for å indikere at den _bare_ blir brukt når man ser Poster-dashboard ☺️

Reply:
Enig!